howto/vscode-debugging-daprd/README.md

```diff
@@ -0,0 +1,150 @@
+# Configuring Visual Studio Code for debugging with daprd
+
+While developing locally in VSCode, you might find it useful to be able to attach the debugger to your code that is utilizing Dapr. To do this, you can run the daprd process separately and then launch your code and attach the debugger. While this is a perfectly acceptable solution, it does require a few extra steps and some instruction to developers who might want to clone your repo and hit the "play" button to begin debugging.
```

Comment:
@lynn-orrell This is great, thanks for adding this.
Could you please clarify here that dapr is the cli executbale and daprd is the runtime executable.
dapr being the cli also generates the components yaml files(if not available) so that it works with the local redis container. daprd will not set up components yaml file, so it must be created by user.